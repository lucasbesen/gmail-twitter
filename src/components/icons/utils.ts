import {css} from "styled-components";

export const sharedStyle = css`
  height: 25px;
  color: #5f6368;
  margin-right: 25px;
  cursor: pointer;
  
  :hover {
    background-color: #eee;
    border-radius: 15px;
  }
`;

export const sidebarIconSharedStyle = css`
  height: 20px;
  color: #767676;
`;
