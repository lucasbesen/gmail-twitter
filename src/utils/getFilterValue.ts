import queryString from "query-string";

type GetFilterValueType = (locationSearch: string) => string;

const getFilterValue: GetFilterValueType = locationSearch => {
  const searchObj = queryString.parse(locationSearch);

  if (searchObj && searchObj.search) {
    const search = searchObj.search.toString();
    return search;
  }
  return '';
};

export default getFilterValue;
