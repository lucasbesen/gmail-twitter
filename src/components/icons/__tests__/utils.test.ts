import { sharedStyle, sidebarIconSharedStyle } from '../utils';

it('should return css for sharedStyle utils', () => {
  expect(sharedStyle).toMatchSnapshot();
});

it('should return css for sidebarIconSharedStyle utils', () => {
  expect(sidebarIconSharedStyle).toMatchSnapshot();
});
