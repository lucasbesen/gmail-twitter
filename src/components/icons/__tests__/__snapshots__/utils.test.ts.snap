// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return css for sharedStyle utils 1`] = `
Array [
  "
  height: 25px;
  color: #5f6368;
  margin-right: 25px;
  cursor: pointer;

  :hover {
    background-color: #eee;
    border-radius: 15px;
  }
",
]
`;

exports[`should return css for sidebarIconSharedStyle utils 1`] = `
Array [
  "
  height: 20px;
  color: #767676;
",
]
`;
